### PR TITLE
Even simpler, even sweeter

### PR DESCRIPTION
**Pull request by alxs**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,13 +1,9 @@
-<img src="https://github.com/w3f/Grants-Program/blob/master/src/Grants_Program.png" style="width:1300px; display: block; margin-left: auto; margin-right: auto;">
```
Comment (semuelle) [inline profile/README.md:1]: What's wrong with a header image?

File: profile/README.md [hunk 2]
```diff
@@ -1,13 +1,9 @@
...
-### Hi there!
+This organisation contains forks of all repositories that have ever been delivered as part of a W3F grant.
+For simplicity, the fork will usually have the same name as the original.
+In the event of a name collision, it will usually be named something like `teamName-repoName`. 
 
-This is the archive of the Web3 Foundation's Grants Program. In this organisation, you will find all repositories 
-that have ever been submitted as part of a grant. For simplicity, the repositories _usually_ have the same name 
-as the original. However, due to the occasional name collision, we might rename the fork to something like 
-`teamName-repoName`. 
+If you cannot find a project you're looking for, please [let us know](mailto:grants@web3.foundation).
```
Comment (semuelle) [inline profile/README.md:7]:
```suggestion
If you cannot find a project you expected here, please [let us know](mailto:grants@web3.foundation).
```

File: profile/README.md (hunk 1 above)
Comment (alxs) [reply profile/README.md:1]: Well, this is not the grants program, it's just the archive. I don't think the banner belongs everywhere related to the grants program but should just be used as a landing banner in the README where it is right now. This place is not a huge thing and doesn't need to look like one.

File: profile/README.md (hunk 2 above)
Comment (alxs) [reply profile/README.md:7]: I'm [not so sure](https://www.google.com/search?q=%22expect+that+here%22&ei=pNjiYvv1AYjVkgWoraHgBA&start=120&sa=N&ved=2ahUKEwj717Oxnpz5AhWIqqQKHahWCEw4bhDy0wN6BAgBEEk&biw=1440&bih=1050&dpr=1) about that formulation. I think [more like](https://books.google.com/ngrams/graph?content=expected+you+here%2Cexpected+to+see+you+here&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cexpected%20you%20here%3B%2Cc0%3B.t1%3B%2Cexpected%20to%20see%20you%20here%3B%2Cc0) "expected to see here", but the original seems much simpler.

File: profile/README.md [hunk 3]
```diff
@@ -1,13 +1,9 @@
-<img src="https://github.com/w3f/Grants-Program/blob/master/src/Grants_Program.png" style="width:1300px; display: block; margin-left: auto; margin-right: auto;">
+Welcome to the Web3 Foundation Grants Program Archive.
 
-### Hi there!
+This organisation contains forks of all repositories that have ever been delivered as part of a W3F grant.
+For simplicity, the fork will usually have the same name as the original.
```
Comment (alxs) [inline profile/README.md]:
```suggestion
For convenience, the fork will usually have the same name as the original.
```

File: profile/README.md (hunk 1 above)
Comment (alxs) [reply profile/README.md:1]: I'll go ahead and merge this. Maybe it doesn't look better without it, we'll see.

Comment (alxs) [reply profile/README.md:1]: I'll go ahead and merge this. Maybe it doesn't look better without it, we'll see.

**Pull request merged**